requirements.txt

```diff
@@ -1,8 +1,12 @@
 fonttools[lxml,unicode,ufo]==3.43.0
 cu2qu==1.6.5
-glyphsLib==4.0.0
+#glyphsLib==4.0.0
```

Comment:
what's missing to use stable glyphsLib version?

Reply:
oh right, this is required
https://github.com/googlefonts/glyphsLib/commit/94d68cf3f79597e1c0312cd740f3532b3847b455

as soon as we merge https://github.com/googlefonts/glyphsLib/pull/538, we can cut a new glyphsLib release. I'd like to have that out by end of week preferably.